src/preload.py

```diff
@@ -836,8 +876,8 @@ def find_one_of(lst, *args):
     return -1
 
 
-def get_docker_storage_driver(image=None):
-    for line in get_docker_service_file_content(image).strip().split("\n"):
+def get_docker_storage_driver(docker_service_file_contents):
+    for line in docker_service_file_contents.strip().split("\n"):
```

Comment:
This is part of an optimization to avoid mounting the rootA partition twice (once for reading `/lib/systemd/system/balena.service` and then again for reading `/etc/issue`). The partition is mounted only once, and contents of those files saved to the `rootA_file_contents` object. The contents of the `balena.service` file are given as the input to this function.